mesonbuild/compilers/fortran.py

```diff
@@ -31,7 +31,9 @@
     PGICompiler
 )
 
-from mesonbuild.mesonlib import EnvironmentException, is_osx, LibType
+from mesonbuild.mesonlib import (
+    EnvironmentException, MachineChoice, is_osx, LibType
```

Comment:
Maybe I'm missing something, but it looks like only `MachineChoice` is used.

Reply:
They're already used. flake8 would get me otherwise.